libs/designsystem/src/lib/components/data-table/_data-table.utils.scss

```diff
@@ -0,0 +1,53 @@
+@use '~@kirbydesign/core/src/scss/utils';
+@use '~@kirbydesign/core/src/scss/interaction-state';
+
+$divider-color: utils.get-color('medium');
+$highlight-color: utils.get-color('light');
+$padding-md: utils.size('s');
+$padding-sm: utils.size('xxs');
+
+@mixin header-typgraphy() {
+  font-weight: utils.font-weight('normal');
+  font-size: utils.font-size('xs');
+  line-height: utils.line-height('xs');
+  color: utils.get-text-color('semi-dark');
+}
+
+@mixin header-padding() {
+  padding: $padding-sm $padding-md;
+}
+
+@mixin header-hover() {
+  &:hover {
+    @include interaction-state.apply-hover;
+  }
+}
+
+@mixin body-typgraphy() {
+  font-weight: utils.font-weight('normal');
+  font-size: utils.font-size('s');
+  line-height: utils.line-height('xs');
+  color: utils.get-text-color('black');
+  vertical-align: middle;
+  min-height: utils.size('xl');
+}
+
+@mixin body-text-padding() {
+  padding: 0 $padding-md;
+}
+
+@mixin body-action-padding() {
+  padding: 0 $padding-sm 0 0;
+}
+
+@mixin body-hover() {
+  &:hover {
+    @include interaction-state.apply-hover;
+
+    background-color: $highlight-color;
+  }
+}
```

Comment:
I agree, maybe there should be an eventEmitter - which "activates" selectable properties of the row, if it is subscribed on.
Another option is, as you mentioned, to have a 'selectable' that can be applied, this would be more appropriate if the assumption is that the consumer will create the 'click-logic' themselves instead of e.g., an exposed eventEmitter.

What are you into?

Reply:
For now, I've gone with solution 2, as you suggested - in the spirit of being as modular as possible.